lib/b_theme.ml

```diff
@@ -95,21 +95,14 @@ let (//) = Filename.concat
 
 (* Some global environment variables *)
 
-module User_dirs = Directories.User_dirs ()
-module Project_dirs = Directories.Project_dirs (struct
-  let qualifier = "org"
-  let organization = "sanette"
-  let application = "bogue"
-end
-)
-
-let home = match User_dirs.home_dir with
-  | None -> failwith "Can't compute home directory path."
-  | Some home -> home
-
-let conf = match Project_dirs.config_dir with
-  | None -> failwith "Can't compute configuration directory path."
-  | Some config_dir -> config_dir
+let xdg = Xdg.create ~env:Sys.getenv_opt ()
+
+let home = Xdg.home_dir xdg
+
+let conf =
+  let application = "bogue" in
+  let config_dir = Xdg.config_dir xdg in
+  Filename.concat config_dir application
```

Comment:
`config_dir // application`

(just to be consistent with the rest)

Reply:
Didn't see that. Now it is changed! Thx